pkg/prowgen/prowgen_konflux.go

```diff
@@ -222,7 +222,7 @@ func GenerateKonflux(ctx context.Context, openshiftRelease Repository, configs [
 					}
 					if len(cfg.ExcludesImages) == 0 {
 						cfg.ExcludesImages = []string{
-							".*source.*",
+							".*-source-.*",
```

Comment:
This is needed as it'd otherwise exclude the aws-ddb-streams-source, aws-s3-source, aws-sqs-source and timer-source.